### DISCUSSION TITLE
Unable to install spacy_grammar

### DISCUSSION THREAD

**Discussion by prachi-ai** [explosion/spaCy#6886] [Help: Other Questions] [answered]
I am getting following message when trying to install spacy_grammar:
"ERROR: No matching distribution found for spacy_grammar"
How can I install it? 
I have used pip install spacy_grammar and conda install spacy_grammar.
In both cases the same error occurs.

**Comment by svlandeg** [comment, marked as answer]
Hi! I'm afraid that's not software owned or written by us. The installation instructions can be found on their GH page: https://github.com/tokestermw/spacy_grammar
